src/operator/nn/fully_connected-inl.h

```diff
@@ -47,7 +48,24 @@ namespace fullc {
 enum FullyConnectedOpInputs {kData, kWeight, kBias};
 enum FullyConnectedOpResource {kTempSpace};
 enum FullyConnectedOpOutputs {kOut};
-}  // fullc
+enum FullyConnectedGradGradOutputs {
```

Comment:
nit: Can we keep the same code style as previous enums

Reply:
Meaning single line?  That's another $50 of cost on CI run for a whitespace change...